mailpoet/tests/_support/AcceptanceTester.php

```diff
@@ -577,8 +580,13 @@ public function selectPaymentMethod($method = 'cod') {
    */
   public function placeOrder() {
     $i = $this;
-    $i->waitForText('Place order');
-    $i->click('Place order');
+    // Add a note to order just to avoid flakiness due to race conditions
```

Comment:
Could you please elaborate on this? Why adding a note avoid flakiness due to race conditions?

Reply:
Sure. This case was also present in WC Core with Playwright, the thing is that when you finish filling customer info, the Place Order button become unclickable for a moment and then it turned to be clickable. We have already wait for element be clickable of course. We can try leaving without writing a note, but in case it's not a problem to leave writing a note then I would proceed with it.

Reply:
I guess it is fine to leave the note.